packages/story-editor/src/utils/test/cleanForSlug.js

```diff
@@ -58,10 +58,15 @@ describe('cleanForSlug', () => {
   });
 
   it('should remove all illegal characters', () => {
-    const result = cleanForSlug('Hello-?,:;!"World');
+    const result = cleanForSlug('Hello-?-,:;!"World');
     expect(result).toBe('hello-world');
   });
 
+  it('should allow using unicode characters', () => {
+    const result = cleanForSlug('హల-వరలడ');
+    expect(result).toBe('హల-వరలడ');
+  });
```

Comment:
Lets add a data set of different slugs and expected return. Maybe test [Cyrillic script](https://en.wikipedia.org/wiki/Cyrillic_script), Arabic, Other [unicode characters](https://en.wikipedia.org/wiki/List_of_Unicode_characters#Latin_Extended_Additional), accented characters like È, hindi characters and mandarin chinese.

Reply:
Added a few more in [7565ca8](https://github.com/GoogleForCreators/web-stories-wp/pull/12603/commits/7565ca80014e8417df5fb0132b745752847b38b0). Accented characters are already handled in a previous test (see the first test of the file).